src/omni-slider.js

```diff
@@ -202,6 +204,21 @@ Slider.prototype.init = function(options) {
   }
 };
 
+Slider.prototype._standardizeUSD_ = function(number) {
+
+  var numberinString = number.toString();
```

Comment:
Sure. I was trying the import/export with my original es2015 version, but 'npm start' failed me. Did you accept MR #3 yet to support es2025?

I am more than happy to refactor to es2015, and per number type as parameter, I will change that to an object to be phase 1.5, as I proposed in another comment.